benches/benchmarks/graphs.rs

```diff
@@ -0,0 +1,116 @@
+use std::collections::HashMap;
+use criterion::{criterion_group, Criterion};
+use oasysdb::db::database::*;
+use crate::benchmarks::utils;
+
+
+fn create_database(path: String, dimension: usize) -> Database {
```

Comment:
Can we move this utility functions to either `benchmarks/mod.rs` along with the `add_values`, `build_graph`, and `search_graph` functions.